configure.ac: fix cross compiling warning

There is a warning while cross compiling which triggered a
failure by do_qa_configure
...
|configure:12652: checking for gzread in -lz
|configure:12677: mips-poky-linux-gcc  -meb -mabi=32 -mhard-float
 -L/lib conftest.c -lz   >&5
|ld: warning: library search path "/lib" is unsafe for cross-compilation
...

While do the lib checking, do not add '-L${Z_DIR}/lib' to LDFLAGS could fix it.

Upstream-Status: Inappropriate [oe specific]
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 configure.ac | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 9a90600..0bac8a4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -391,8 +391,6 @@ if test "$with_zlib" = "no"; then
     echo "Disabling compression support"
 else
     AC_CHECK_HEADERS(zlib.h,
-        [SAVE_LDFLAGS="${LDFLAGS}"
-	 LDFLAGS="-L${Z_DIR}/lib"
 	AC_CHECK_LIB(z, gzread,[
 	    AC_DEFINE([HAVE_LIBZ], [1], [Have compression library])
 	    WITH_ZLIB=1
@@ -406,8 +404,7 @@ else
 		esac]
 	    else
 		Z_LIBS="-lz"
-	    fi])
-	 LDFLAGS="${SAVE_LDFLAGS}"])
+	    fi]))
 fi
 
 AC_SUBST(Z_CFLAGS)
-- 
1.9.1

